nextflow_schema.json

```
@@ -15,12 +15,12 @@
                 "input": {
                     "type": "string",
                     "format": "file-path",
-                    "mimetype": "text/csv",
-                    "pattern": "^\\S+\\.(yaml|yml)$",
+                    "mimetype": "text/yaml",
+                    "pattern": "^\\S+\\.yaml$",
                     "schema": "assets/schema_input.json",
-                    "description": "Path to YAML file containing information about the samples in the experiment.",
+                    "description": "Path to comma-separated file containing information about the samples in the experiment.",
```

```suggestion
                    "description": "Path to YAML file containing information about the samples in the experiment.",
```
You can use the `nf-core schema build` tool to check the rest of the document via the online GUI.